### ISSUE TITLE
Computed property is firing warning on private members

### ISSUE THREAD

**Issue by HitalloExiled** [Microsoft/TypeScript#20929]
**TypeScript Version:**  2.7.0-dev.20171226

**Code**

```ts
const foo = Symbol.for("foo");
const baz = Symbol.for("baz");

export class Bar
{
    private [foo]: string;
          // ^^^ '__computed' is declared but its value is never read.
    public  [baz]: string; // OK

    public constructor()
    {
        this[foo] = "foo";
        let foo$ = this[foo];
        foo$.toString();
    }
}
```

**Expected behavior:**
No warnings
**Actual behavior:**
Incorrect warnings
